Generator/BotBuilderVSIX-V4/ProjectTemplates/Bot Framework/AspNetCore-EchoBot-With-State.csproj

```diff
@@ -0,0 +1,14 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.AspNetCore.All" Version="2.0.5" />
+    <PackageReference Include="Microsoft.Bot.Builder.Core" Version="4.0.0-alpha-m3.1" />
```

Comment:
This is great, but please know we'll have to upgrade these at least once prior to Build.

Reply:
Acknowledged! Thanks!